### ISSUE TITLE
Vid2depth pre trained model doubt

### ISSUE THREAD

**Issue by vinayver198** [tensorflow/models#10099]
Hi @rezama ,
I have a question related to model available on vid2depth project website . Is the model available trained with icp loss or without icp loss ?

**Comment by rezama** [comment]
With icp loss.

**Comment by xyn123321** [comment]
Hi @rezama ,
I wonder how to draw dense gt depth map with kitti dataset ? Like the gt depth maps in your paper.
![QQ截图20211015102830](https://user-images.githubusercontent.com/44809132/137424264-916f905e-af21-4a60-8100-33042de61292.jpg)
I read some papers, they mentioned that the ground truth velodyne depth being very sparse, authors interpolate it for visualization purposes. But I can't figure out how to do that.
I will be very appreciated if you can help! I'm eager for your reply. Thanks!

**Comment by rezama** [comment]
See this function:
https://github.com/tinghuiz/SfMLearner/blob/master/kitti_eval/depth_evaluation_utils.py#L171

**Comment by ZhuYingJessica** [comment]
@xyn123321  Hi, I wonder if you have solved the dense GT depth map problem? I met the same problem as you. Could you please send the drawn depth maps to me if you have drawn them? I do appreciate it!
